docs/config-janitor.rst

```diff
@@ -1,5 +1,5 @@
-Configuration
-=============
+Janitor Configuration
+=====================
```

Comment:
Without this differentiation, the ToC would be this:

![2018-06-01 at 4 08 pm](https://user-images.githubusercontent.com/1191069/40862298-100fdf06-65ba-11e8-9ddc-9c75e49e1c01.png)